script.js

```diff
@@ -0,0 +1,120 @@
+    /* eslint-disable */
+    function applyFunction() {
+      let testString = document.getElementById("testString").value;
+      let selectMethod = document.getElementById("selectMethod").value;
+      let error = '';
+      let result;
+      let methodDetails = '';
```

Comment:
@andela-kakpobome I just addressed this with the PR:  https://github.com/andela-sayidu/string-class/pull/4/files